client/components/bulk-select/docs/example.jsx

```diff
@@ -57,7 +57,9 @@ module.exports = React.createClass( {
 					<a href="/devdocs/design/bulk-selects">BulkSelects</a>
 				</h2>
 				<Card>
-					<BulkSelect totalElements={ this.state.elements.length } selectedElements={ this.getSelectedElementsNumber() } onToggle={ this.handleToggleAll } />
+					<div>
```

Comment:
yuuuuuup, exactly! I changed the bulk-select component to be a inline-block one by default, so this div is just to force a new line :)